src/sentry/tasks/relay.py

```diff
@@ -211,7 +224,7 @@ def invalidate_project_config(
     These will be addressed in the future using config revisions tracked in Redis.
     """
     validate_args(organization_id, project_id, public_key)
-    sentry_sdk.set_tag("trigger", trigger)
+    sentry_sdk.set_tag("update_reason", trigger)
```

Comment:
Tags use `update_reason` as key. We may want to change it to `trigger`, but that's out of scope for this PR. For now, let's be consistent and stick to `update_reason`.